rse/portfolio.rst

```diff
@@ -101,22 +101,22 @@ Julia is a relatively new programming language that has found many
 users in certain fields.  A professor taught an undergraduate course
 using Julia, but there were not sufficient introductory resources to
 prepare students for the course, nor any other resources to prepare
-them.  Aalto RSE find an open-source course prepared by CSC (Finland's
-national scientific computing center), improved it to handle the
+them.  Aalto RSE found an open-source course prepared by CSC (Finland's
+National Scientific Computing Center), improved it to handle the
```

Comment:
```suggestion
national scientific computing center), improved it to handle the
```
I think this should actually stay lower case since it's just a description of CSC, not the actual full name

Reply:
I agree.
Done.